samples/gibbs_ensemble/gibbs.py

```diff
@@ -0,0 +1,139 @@
+import socket
+import struct
+import pickle
+import numpy as np
+
+
+class MessageId:
```

Comment:
this should use enumerations which are a language feature since 3.4:

``` python
import enum

@enum.unique
class MessageId(enum.Enum):
    START = 0
```